overlord/servicestate/quota_handlers.go

```diff
@@ -560,6 +563,61 @@ type ensureSnapServicesForGroupOptions struct {
 	extraSnaps []string
 }
 
+func snapServiceNames(info *snap.Info) []string {
+	var appNames []string
+	for _, app := range info.Services() {
+		appNames = append(appNames, fmt.Sprintf("%s.%s", info.InstanceName(), app.Name))
+	}
+	return appNames
+}
+
+// affectedSnapServices returns a map of snaps and the services affected
+// by performing changes to a quota group. For groups that contain just snaps, all
+// services are added to the map, for groups that contain specific services, only those
+// services are affected.
```

Comment:
as written the comment is not very clear on the fact that if there are services in the group there cannot be snaps

Reply:
I'll add a note of this!